src/components/flowchart-primary-toolbar/flowchart-primary-toolbar.test.js

```diff
@@ -31,14 +38,15 @@ describe('PrimaryToolbar', () => {
     const wrapper = setup.mount(<ConnectedFlowchartPrimaryToolbar />, {
       visible,
     });
-    expect(wrapper.find('.pipeline-icon-toolbar__button').length).toBe(3);
+    expect(wrapper.find('.pipeline-icon-toolbar__button').length).toBe(4);
```

Comment:
Not a big thing, I'm thinking maybe rather using the number, 3 or 4, we can use `visible.length`? just so we dont have to update again if we add another button